src/meta/test/JobManagerTest.cpp

```diff
@@ -13,6 +13,8 @@
 #include "kvstore/Common.h"
 #include "webservice/WebService.h"
 
+#define private public
```

Comment:
What's this.....

Reply:
black magic...
this will make test code modify private member directly. 
without declare any "FRIEND_TEST"